.gitignore

```diff
@@ -6,3 +6,5 @@ _site/
 build/
 dist/
 whotrackme.egg-info/
+static/css/
```

Comment:
Shouldn't we generate the `css/` in the `_site` folder directly? Which is already ignored.